src/main/java/com/ea/restaurant/etl/MongoToPostgresqlOrderStatusHistory.java

```diff
@@ -0,0 +1,72 @@
+package com.ea.restaurant.etl;
+
+import com.ea.restaurant.constants.UserType;
+import com.ea.restaurant.document.MongoOrderStatusHistory;
+import com.ea.restaurant.entities.OrderStatusHistory;
+import com.ea.restaurant.service.MongoOrderStatusHistoryService;
+import com.ea.restaurant.service.OrderStatusHistoryService;
+import com.ea.restaurant.util.OrderStatusHistoryMapper;
+import com.ea.restaurant.util.OrderStatusHistoryUtil;
+import java.util.ArrayList;
+import java.util.List;
+import org.bson.types.ObjectId;
+import org.springframework.scheduling.annotation.Scheduled;
+
+public class MongoToPostgresqlOrderStatusHistory {
```

Comment:
Could you create a class to abstract how an ETL works?

Example:
```
public interface Etl<S, T> {

  S extract();
  
 T extract(S s);

 void load(T t);

}
```

Every ETL should implement this interface.

Reply:
Done.